apps/web/pages/auth/setup.tsx

```diff
@@ -0,0 +1,285 @@
+                type="email"
+                name="email"
+                autoCapitalize="none"
+                autoComplete="email"
+                autoCorrect="off"
+                placeholder="Email address"
+                className={classNames(
+                  "rounded-r-s mt-0 block min-w-0 flex-1 rounded-none border-gray-300 px-3 py-2 sm:text-sm",
+                  formMethods.formState.errors.email
+                    ? "border-r-0 focus:border-l focus:border-gray-300 focus:ring-0"
+                    : "focus:border-gray-900 focus:ring-gray-900"
+                )}
+              />
+            )}
+          />
+        </div>
+        {formMethods.formState.errors.email && (
+          <p role="alert" className="mt-1 text-xs text-red-500">
+            {formMethods.formState.errors.email.message}
+          </p>
+        )}
+      </div>
+      <div>
+        <label htmlFor="password" className="sr-only">
+          Password
+        </label>
+        <div
+          className={classNames(
+            "flex rounded-sm",
+            formMethods.formState.errors.password ? "border-2 border-red-500" : "border-0"
+          )}>
+          <Controller
+            name="password"
+            control={formMethods.control}
+            render={({ field: { onBlur, onChange, value } }) => (
+              <Input
+                value={value || ""}
+                onBlur={onBlur}
+                onChange={async (e) => {
+                  onChange(e.target.value);
+                  formMethods.setValue("password", e.target.value);
+                  await formMethods.trigger("password");
+                }}
+                color={formMethods.formState.errors.password ? "warn" : ""}
+                type="password"
+                name="password"
+                autoComplete="off"
+                placeholder="Password"
+                className={classNames(
+                  "rounded-r-s mt-0 block min-w-0 flex-1 rounded-none border-gray-300 px-3 py-2 sm:text-sm",
+                  formMethods.formState.errors.password
+                    ? "border-r-0 focus:border-l focus:border-gray-300 focus:ring-0"
+                    : "focus:border-gray-900 focus:ring-gray-900"
+                )}
+              />
+            )}
+          />
+        </div>
+        {formMethods.formState.errors.password && (
+          <p role="alert" className="mt-1 max-w-xs text-xs text-red-500">
+            {formMethods.formState.errors.password.message}
+          </p>
+        )}
+      </div>
+
+      <input type="submit" id="submit-setup-step-1" className="hidden" />
+    </Form>
+  );
+};
+
+export default function Setup(props: inferSSRProps<typeof getServerSideProps>) {
+  const steps = [
+    {
+      title: "Administrator user",
+      description: "Let's create the first administrator user.",
+      content: props.userCount !== 0 ? <StepDone /> : <SetupFormStep1 />,
+      enabled: props.userCount === 0,
```

Comment:
This is to check if the wizard should show buttons to navigate through more steps.